src/auto_archiver/modules/gsheet_feeder_db/gsheet_feeder_db.py

```diff
@@ -120,14 +130,26 @@ def done(self, item: Metadata, cached: bool = False) -> None:
         gw, row = self._retrieve_gsheet(item)
         # self._safe_status_update(item, 'done')
 
+        # DM - success log message showing the row, sheet and tab
+        spreadsheet = gw.wks.spreadsheet.title
+        worksheet = gw.wks.title
+        logger.success(f" row {row} on {spreadsheet} : {worksheet}")
```

Comment:
any particular reason for this?

Reply:
I look at logs every day, and this helps me a lot to see which row in the spreadsheet and tab, a problem is on (rather than an entry number). Just a nice to have for me.

Reply:
ok, adding this do /dev